packages/web-components/src/components/va-minimal-footer/va-minimal-footer.scss

```diff
@@ -3,7 +3,7 @@
 :host {
   display: flex;
   justify-content: center;
-  background-color: var(--color-primary-darkest);
+  background-color: var(--color-primary-darker);
```

Comment:
Shifts up a level due to names and values being changed per [this comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069)